CMakeLists.txt

```diff
@@ -49,19 +49,41 @@ if(GETTEXT_FOUND)
     # translations target will convert .po files into .mo and .qm as needed.
     # The files are checked for a _qt suffix and if it is found, converted to
     # qm, otherwise they are converted to .po.
-    add_custom_target(translations ALL)
-    # copy-translations can be used to copy the built translation files from the
-    # build directory to the source resources directory. This is mostly a convenience
-    # during development, normally you want to simply use the install target to install
-    # the files along side the rest of the application.
-    add_custom_target(copy-translations)
+    add_custom_target(i18n-create-po)
+    add_custom_target(i18n-update-po)
+    add_custom_target(i18n-create-mo ALL)
 
     SUBDIRLIST(languages ${CMAKE_SOURCE_DIR}/resources/i18n/)
-    foreach(lang ${languages})
-        file(GLOB po_files ${CMAKE_SOURCE_DIR}/resources/i18n/${lang}/*.po)
+    file(GLOB pot_files ${CMAKE_SOURCE_DIR}/resources/i18n/*.pot)
+    foreach(language ${languages})
+        # *.po files
+        add_custom_target(i18n-create-po-${language})
```

Comment:
It would be nice if we can move all this code into its own macro, since we now create individual targets for each translation.